examples/aws_orchestrated/README.md

```diff
@@ -0,0 +1,107 @@
+# Run pipelines on the cloud cheaply
+You can easily run zenml pipelines on a cloud VM instance if local compute is not powerful enough. With this ability, it 
+is simple to run on cheap preemptible/spot instances to save costs.
+
+## Adding an orchestration backend to a pipeline
+The pattern to add a backend to the pipeline is:
+
+```python
+backend = ...  # define the orchestrator backend you want to use
+pipeline.run(backend=backend)  # you can also do this at construction time
+```
+
+## Running a pipeline a AWS-EC2 Instance
+This example utilizes [Amazon Elastic Compute Cloud (EC2)](https://aws.amazon.com/ec2/) to launch an EC2 instance on 
+ AWS, which then runs the specified pipeline.
+
+### Pre-requisites
+In order to run this example, you need to clone the zenml repo.
+
+```bash
+git clone https://github.com/maiot-io/zenml.git
+```
+
+Before continuing, either [install the zenml pip package](https://docs.zenml.io/getting-started/installation.html) or 
+install it [from the cloned repo](../../zenml/README.md). In both cases, make sure to also install the 
+`aws` extension (e.g. with pip: `pip install zenml[aws]`)
+
+```
+zenml init
```

Comment:
There is a small error here across the examples folder. The init should happen AFTER doing the first `cd zenml` , otherwise it tries to do a zenml init to the outer folder. Can we fix it here and ill fix it on main for the others? Or you can fix all example READMEs together if you'd like. Either way, we should do it now

Reply:
Update: I updated all the other examples in the `main` branch

Reply:
I am on it!

Reply:
I have updated this branch!